kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
actually, looks like it already happened again: https://travis-ci.org/chef/chef/jobs/151909377